frontend/templates/views/partials/author.j2

```diff
@@ -7,6 +7,19 @@
   <ul class="ap-o-mini-card-list">
     <li class="ap-o-mini-card-list-item">
       <div class="ap-m-mini-card">
+        {% if doc.author is iterable and doc.author is not string %}
```

Comment:
You can avoid duplicating the amp-img by converting the author from a string to a list.

Reply:
I just didn't know a good way to do that! Thanks for providing one :)